src/main/kotlin/extensions/yuan/varenyzc/NeedNonNullableClassesSupport.kt

```diff
@@ -0,0 +1,76 @@
+package extensions.yuan.varenyzc
+
+import com.intellij.util.ui.JBDimension
+import extensions.Extension
+import wu.seal.jsontokotlin.model.classscodestruct.DataClass
+import wu.seal.jsontokotlin.model.classscodestruct.KotlinClass
+import wu.seal.jsontokotlin.ui.*
+import javax.swing.JPanel
+
+object NeedNonNullableClassesSupport : Extension() {
+
+    const val prefixKeyEnable = "top.varenyzc.need_nonnullable_classes_enable"
+    const val prefixKey = "top.varenyzc.need_nonnullable_classes"
+
+    override fun createUI(): JPanel {
+
+        val prefixJField = jTextInput(getConfig(prefixKey), getConfig(prefixKeyEnable).toBoolean()) {
+            addFocusLostListener {
+                if (getConfig(prefixKeyEnable).toBoolean()) {
+                    setConfig(prefixKey, text)
+                }
+            }
+            maximumSize = JBDimension(400, 30)
+        }
+
+        return jHorizontalLinearLayout {
+            jCheckBox("Need NonNullable classes: ", getConfig(prefixKeyEnable).toBoolean(), { isSelected ->
```

Comment:
For this name, I really don't know what it does until I see your test case😂, Try to think of a better name and paste a link to the UI to show the usage and demo

Reply:
😂,okay, i will think about it.Maybe my English is so poor.